docs/appendices/release-notes/unreleased.rst

```diff
@@ -61,6 +61,10 @@ Breaking Changes
   execution is only available via the ``results`` array represented by a
   row count for each bulk operation.
 
+- Numeric literals fitting into the ``integer`` range will be detected now as
+  ``integer`` literals instead of ``bigint`` literals. Thus a statement like
+  ``select 1`` will return an ``integer`` column type.
```

Comment:
Maybe also elaborate that for most clients this shouldn't be an issue - since for HTTP it was JSON serialized, and for PostgreSQL clients there is often a typed `getLong`, etc..

Reply:
Added your suggestion, please check.